lynx/src/main/java/com/github/pedrovgs/lynx/LynxShakeDetector.java

```diff
@@ -46,6 +48,9 @@ public void init() {
       @Override public void hearShake() {
         if (isEnabled) {
           openLynxActivity();
+          if (keepAlwaysActive) {
+            disable();
+          }
```

Comment:
Instead of enable and disable the component why don't we just check if the application is in background before to open ``LynxActivity``?